work9/index.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const data = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク",
+},
+{
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+},
+];
+
+function getData() {
+  const result = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(data);
+
+    }, 3000);
+  })
+  return result;
+}
+
+async function outputFunction() {
```

Comment:
fix function name

Reply:
Changed the function name.

Please check.


dbfd9e2

Reply:
include loading function

Reply:
included loadin function into displayView function.

Please check.

fb34305